ngraph/frontend/paddlepaddle/src/model.cpp

```diff
@@ -210,35 +244,37 @@ namespace ngraph
             }
         }
 
-        InputModelPDPD::InputModelPDPDImpl::InputModelPDPDImpl(const std::string& path,
+        InputModelPDPD::InputModelPDPDImpl::InputModelPDPDImpl(const path_type& path,
                                                                const InputModel& input_model)
             : m_fw_ptr{std::make_shared<ProgramDesc>()}
             , m_input_model(input_model)
         {
-            std::string ext = ".pdmodel";
-            std::string model_file(path);
+            path_type model_file{path};
             std::unique_ptr<std::ifstream> weights_stream;
-            if (model_file.length() >= ext.length() &&
-                (0 == model_file.compare(model_file.length() - ext.length(), ext.length(), ext)))
+            if (model_file.length() >= pdpd::EXT.length() &&
+                (0 == model_file.compare(
+                          model_file.length() - pdpd::EXT.length(), pdpd::EXT.length(), pdpd::EXT)))
             {
-                std::string weights_file(path);
-                weights_file.replace(weights_file.size() - ext.size(), ext.size(), ".pdiparams");
+                path_type weights_file{path};
+                weights_file.replace(
+                    weights_file.size() - pdpd::EXT.size(), pdpd::EXT.size(), pdpd::PARAMS_EXT);
                 weights_stream = std::unique_ptr<std::ifstream>(
                     new std::ifstream(weights_file, std::ios::binary));
                 // Don't throw error if file isn't opened
                 // It may mean that model don't have constants
             }
             else
             {
-                model_file += "/__model__";
+                model_file += pdpd::MODEL_NAME;
```

Comment:
Should there be separator? \ or /

Reply:
separator is embedded in the pdpd::MODEL_NAME